puppet/modules/chassis/lib/puppet/parser/functions/sz_extensions.rb

```diff
@@ -1,15 +1,15 @@
 require "/vagrant/puppet/chassis"
 
 module Puppet::Parser::Functions
+	newfunction(:chassis_get_global_extensions, :type => :rvalue) do |args|
```

Comment:
Did you move this one to get them in alphabetical order?

Reply:
This is to avoid an issue:

```
    default: Error: Evaluation Error: Error while evaluating a Function Call, The code loaded from /vagrant/puppet/modules/chassis/lib/puppet/parser/functions/sz_extensions.rb produced mis-matched name, expected 'function_sz_extensions', got 'function_chassis_get_global_extensions' (file: /vagrant/puppet/manifests/development.pp, line: 2, column: 15) on node siemens.local
```

My well honed instincts in this area lead me to the specific reasoning that I should try random shit until it works and hope for the best, this is what worked!

Reply:
That's a weird one! No worries. It won't affect anything for other providers so I'm happy to leave this then!